app/pages/patientnew/patientnew.js

```diff
@@ -128,42 +142,36 @@ export let PatientNew = translate()(React.createClass({
         ),
       })
     }
-  },
-
-  getInitialState: function() {
-    return {
-      working: false,
-      formValues: {
-        isOtherPerson: false,
-        fullName: this.getUserFullName(),
-        dataDonateDestination: ''
-      },
-      validationErrors: {},
-    };
-  },
+  }
 
-  componentDidMount: function() {
+  componentDidMount() {
     if (this.props.trackMetric) {
       this.props.trackMetric('Viewed Profile Create');
     }
-  },
+  }
 
-  componentWillReceiveProps: function(nextProps) {
+  componentWillReceiveProps(nextProps) {
     this.setState({
       formValues: _.assign(this.state.formValues, {
-        fullName: this.getUserFullName(nextProps)
+        firstName: this.getUserFirstName(nextProps),
+        lastName: this.getUserLastName(nextProps)
       })
     });
-  },
+  }
+
+  getUserFirstName(props) {
+    props = props || this.props;
+    return personUtils.firstName(props.user) || '';
+  }
 
-  getUserFullName: function(props) {
+  getUserLastName(props) {
     props = props || this.props;
-    return personUtils.fullName(props.user) || '';
-  },
+    return personUtils.lastName(props.user) || '';
```

Comment:
personUtils.lastName() & personUtils.firstName() should return empty string by default.